### PR TITLE
Add hackmd note to march meetup page

### PR DESCRIPTION
**Pull request by tychuang1211**

### CODE REVIEW

File: content/pages/meetup/2023/03-nycu.rst [hunk 1]
```diff
@@ -37,6 +37,15 @@ Venue
     </iframe>
   </div>
 
+Meetup note
```
Comment (yungyuc) [inline content/pages/meetup/2023/03-nycu.rst:40]: Could you add a link targeting this somewhere former in the document?

**Review by yungyuc — CHANGES_REQUESTED** [review]
Something seems wrong with netlify.  Please also try to see if it's a real issue.  cc @chestercheng for his awareness.

**Comment by chestercheng** [comment]
> Something seems wrong with netlify. Please also try to see if it's a real issue. cc @chestercheng for his awareness.

This is the network issue of Netlify, I have manually retry this build.
preview url: https://641267fcf5fcd502bd52662d--swportal.netlify.app

**Comment by yungyuc** [comment]
@tychuang1211 when there's time could you please make an update to the PR or let us know if there's any issues?

**Comment by yungyuc** [comment]
Thanks @tychuang1211

**Pull request merged**